src/RA_Dlg_Memory.cpp

```diff
@@ -615,23 +614,23 @@ void MemoryViewerControl::RenderMemViewer(HWND hTarget)
                             g_MemBookmarkDialog.WriteFrozenValue(*bm);
                     }
                 }
+                std::array<unsigned char, 16> data{};
```

Comment:
this should be allocated once outside the loop

Reply:
I was looking through this again and could've sworn I've done that after you suggested this. It wasn't but now it is.